stock_secondary_unit_weighing/models/stock_move_line.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    has_recorded_weight = fields.Boolean(help="The weight was set from the wizard")
+    recorded_weight = fields.Float(digits="Product Unit of Measure")
+    weighing_user_id = fields.Many2one(comodel_name="res.users")
+    weighing_date = fields.Datetime()
```

Comment:
I don't get how this is tied to the secondary unit

Reply:
It hasn't. It just slipped into the code and it overwriting the original fields. Thanks for pointing out